packages/core/src/login/webview/vue/login.vue

```diff
@@ -308,9 +312,16 @@ export default defineComponent({
                     }
                 } else if (this.selectedLoginOption === LoginOption.ENTERPRISE_SSO) {
                     this.stage = 'SSO_FORM'
-                } else if (this.selectedLoginOption === LoginOption.EXISTING_LOGINS) {
-                    // TODO:
-                    this.stage = 'START'
+                } else if (this.selectedLoginOption >= LoginOption.EXISTING_LOGINS) {
+                    const selectedConnection =
+                        this.existingLogins[this.selectedLoginOption - LoginOption.EXISTING_LOGINS]
```

Comment:
why are we doing math on the selected options/connections?

Reply:
This is to handle multiple existing connections of toolkit. I can refactor in future commits. (not in this PR)